tiling.js

```diff
@@ -352,7 +355,7 @@ export class Space extends Array {
         this.setSpaceTopbarElementsVisible();
 
         // apply default focus mode
-        setFocusMode(getDefaultFocusMode(), this);
+        setFocusMode(focusMode ?? getDefaultFocusMode(), this);
```

Comment:
Is the variable needed for functionality, or would this have the same effect?
```suggestion
        setFocusMode(prevSpace?.focusMode ?? getDefaultFocusMode(), this);
```
I'd find ^that slightly more readable, since I don't need to glance up at an earlier line to see where `focusMode` comes from.

Reply:
Hey @smichel17,

Yes, this was done due to a concern of a race condition from the `saveState` deletion and setting the focus mode after `this.setSpaceTopbarElementsVisible();`.

Logically I wanted to maintain current/proven/tested order of restoring state, e.g. 

_get/do all our `prevState` things_ ==> _delete/finish prevState usage_ ==> _set space topbar element initial visibility_ ==> _set focus mode_.

Having said that, the concern with a race condition is probably overblown, and thinking about it, we could probably replace line 348 with ` setFocusMode(prevSpace?.focusMode ?? getDefaultFocusMode(), this);`.

Feel free to test and run it for a while and let me know if there's any issues!